js/ui/collection/ui.collection_widget.edit.js

```diff
@@ -481,18 +481,20 @@ var CollectionWidget = BaseCollectionWidget.inherit({
     },
 
     _postprocessRenderItem: function(args) {
-        if(this.option("selectionMode") === "none") {
-            return;
+        if(this.option("selectionMode") !== "none") {
+            var $itemElement = $(args.itemElement),
+                normalizedItemIndex = this._editStrategy.getNormalizedIndex($itemElement),
+                isItemSelected = this._isItemSelected(normalizedItemIndex);
+
+            this._processSelectableItem($itemElement, isItemSelected);
         }
+    },
 
-        var $itemElement = $(args.itemElement);
+    _processSelectableItem: function($itemElement, isSelected) {
+        var selectedItemClass = this._selectedItemClass();
 
-        if(this._isItemSelected(this._editStrategy.getNormalizedIndex($itemElement))) {
-            $itemElement.addClass(this._selectedItemClass());
-            this._setAriaSelected($itemElement, "true");
-        } else {
-            this._setAriaSelected($itemElement, "false");
-        }
+        isSelected ? $itemElement.addClass(selectedItemClass) : $itemElement.removeClass(selectedItemClass);
```

Comment:
Maybe better to use `$itemElement.toggleClass(this._selectedItemClass(), isSelected)`  ?